services/test.http

```diff
@@ -1,9 +1,9 @@
-POST http://localhost:3030/user
+POST http://localhost:3030/api/user
 Content-Type: application/json
 
 {
-  "email": "foo@example.com",
-  "password": "secret123",
+  "email": "fooxample.com",
+  "password": "se3",
```

Comment:
Did you intend to commit this?

Reply:
5cb7a0f1d944f31e6f219bd04f3ecf3115f4280a